packages/protocol/docs/Derivation.md

```diff
@@ -527,7 +527,7 @@ The anchor transaction executes a carefully orchestrated sequence of operations:
 
 1. **Fork validation and duplicate prevention**
 
-   - Verifies the current block number is at or after the Shasta fork height
+   - Verifies the current block timestamp is at or after the Shasta fork activation timestamp
    - Tracks parent block hash to prevent duplicate `anchorV4` calls within the same block
```

Comment:
I think the `parent block hash` here is the `blockState.ancestorsHash`? And actually duplicate `anchorV4` calls within the same block is allowed in current consensus, but only the first call in a block won't revert.